doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,11 +8,18 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
+:class:`~ray.train.ScalingConfig` overview
```

Comment:
I think we repeat this API reference way too many times:
1. In the previous line
2. In the header
3. In the first sentence
4. As a `seealso` tip at the end of the section.

I think we can:

1. Just rename this section to "Overview"
2. Remove the `seealso` block now.